### PR TITLE
formatting

### PR DESCRIPTION
**Pull request by mowangjuanzi**
According to phpstorm prompt

### CODE REVIEW

File: src/Worker.php [hunk 1]
```diff
@@ -1462,9 +1459,6 @@ protected static function forkOneWorkerForLinux(self $worker)
     {
         // Get available worker id.
         $id = static::getId($worker->workerId, 0);
-        if ($id === false) {
```
Comment (mowangjuanzi) [inline src/Worker.php:1465]: because `$id`'s return type is int. so  `$id === false` is never equal.

File: src/Worker.php [hunk 2]
```diff
@@ -1047,7 +1044,7 @@ protected static function formatStatusData($statistics_file)
         $total_timers = 0;
         $maxLen1 = static::$_maxSocketNameLength;
         $maxLen2 = static::$_maxWorkerNameLength;
-        foreach($info as $key => $value) {
```
Comment (mowangjuanzi) [inline src/Worker.php:1050]: `$key` is not used.

File: src/Worker.php [hunk 3]
```diff
@@ -1003,9 +1002,7 @@ protected static function parseCommand()
                 \posix_kill($master_pid, $sig);
                 exit;
             default :
-                if (isset($command)) {
```
Comment (mowangjuanzi) [inline src/Worker.php:1006]: `$command` never is `null`. because line 886.

File: src/Worker.php [hunk 4]
```diff
@@ -972,7 +971,7 @@ protected static function parseCommand()
                 $start_time = \time();
                 // Check master process is still alive?
                 while (1) {
-                    $master_is_alive = $master_pid && \posix_kill((int) $master_pid, 0);
```
Comment (mowangjuanzi) [inline src/Worker.php:975]: `$master_pid` is already `int`

**Comment by walkor** [comment]
Thanks.

**Pull request merged**